packages/server/src/controllers/Controller.js

```diff
@@ -111,6 +112,68 @@ export class Controller {
   }
 
   setupAction(type, name, handler, authorize) {
+    if (!isFunction(handler)) {
+      const setupHandlerFromParam = ({
```

Comment:
It doesn't look like this inner function is actually accessing anything from the outer scope? In that case, the dito.js way has been to declare it as an oldfashioned priveate function outside of the class at the end of the file, as `function setupHandlerFromParam()`. And I'd suggest to call it `setupHandlerFromObject()` instead?